tests/e2e/local_interactive_sdk_oauth_test.py

```diff
@@ -0,0 +1,73 @@
+from codeflare_sdk import Cluster, ClusterConfiguration, TokenAuthentication, generate_cert
+
+import math
+import pytest
+import ray
+
+from support import *
+
+@pytest.mark.openshift
+class TestRayLocalInteractiveOauth:
```

Comment:
Would it be possible to create another test without oauth to be executed against KinD?